service/history/timerQueueStandbyTaskExecutor.go

```diff
@@ -272,6 +278,49 @@ func (t *timerQueueStandbyTaskExecutor) executeActivityTimeoutTask(
 	)
 }
 
+func (t *timerQueueStandbyTaskExecutor) executeActivityRetryTimerTask(
+	ctx context.Context,
+	task *tasks.ActivityRetryTimerTask,
+) (retError error) {
+
+	actionFn := func(context workflow.Context, mutableState workflow.MutableState) (interface{}, error) {
+
+		activityInfo, ok := mutableState.GetActivityInfo(task.EventID) //activity schedule ID
+		if !ok {
+			return nil, nil
+		}
+
+		ok = VerifyTaskVersion(t.shard, t.logger, mutableState.GetNamespaceEntry(), activityInfo.Version, task.Version, task)
+		if !ok {
+			return nil, nil
+		}
+
+		if activityInfo.Attempt > task.Attempt {
```

Comment:
I realize we have the same thing in timer active execution. But I am curious why we use `>` instead of `==`?